client/my-sites/site-settings/form-general.jsx

```diff
@@ -609,10 +617,33 @@ export class SiteSettingsFormGeneral extends Component {
 
 		const LaunchCard = showPreviewLink ? CompactCard : Card;
 
+		if ( isSiteOnECommerceTrial ) {
+			const eCommerceTrialUpsellText = translate(
+				'Before you can share your store with the world, you need to {{NoticeAction}}pick a plan{{/NoticeAction}}.',
+				{
+					components: {
+						NoticeAction: (
+							<NoticeAction href={ `/plans/${ siteSlug }?plan=ecommerce-bundle-monthly` } />
+						),
+					},
+				}
+			);
```

Comment:
Given that we've switched to inline text for the CTA, we should use a more standard `a` tag, and not a `NoticeAction`. If you take a look at the [Notice dev docs](https://wpcalypso.wordpress.com/devdocs/design/notice), the `NoticeAction` is intended for a separate CTA (which we had before), and not an inline link.
```suggestion
			// TODO: flesh this out and wire it up, but we want to be able to understand how often people click on the CTA
			// We should also trigger an event when the notice is rendered - can we do that via a functional wrapper component that uses `useEffect()` to trigger a render based on the incoming props?
			const recordTracksEventForClick = recordTracksEvent( 'calypso_ecommerce_trial_launch_banner_click' );
			const eCommerceTrialUpsellText = translate(
				'Before you can share your store with the world, you need to {{a}}pick a plan{{/a}}.',
				{
					components: {
						a: (
							<a href={ `/plans/${ siteSlug }?plan=ecommerce-bundle-monthly` } onClick={ recordTracksEventForClick }  />
						),
					},
				}
			);
```